solutions/src/test/java/ru/sberbank/school/task08/SerializationTest.java

```diff
@@ -0,0 +1,66 @@
+package ru.sberbank.school.task08;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import ru.sberbank.school.task08.state.GameObject;
+import ru.sberbank.school.task08.state.InstantiatableEntity;
+import ru.sberbank.school.task08.state.MapState;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Created by Mart
+ * 03.06.2019
+ **/
+public class SerializationTest {
+    private String fileName = "saved.tmp";
+    private MapState<GameObject> saved;
+    private String directoryPath = "D:\\";
+
+    @BeforeEach
+    void init() {
+            List<GameObject> list = new ArrayList<>();
+            list.add(new GameObject(InstantiatableEntity.Type.BUILDING, InstantiatableEntity.Status.DESPAWNED, 500));
+            list.add(new GameObject(InstantiatableEntity.Type.ENEMY, InstantiatableEntity.Status.KILLED, 0));
+            saved = new MapState<>("mapState", list);
+    }
+
+    @Test
+    void SerializableManagerTest() {
+        try {
+            SerializableManager manager = new SerializableManager(directoryPath);
+            manager.saveGame(fileName, saved);
+            MapState<GameObject> loaded = manager.loadGame(fileName);
+            assertEquals(saved, loaded);
+        } catch (SaveGameException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    @Test
+    void KryoManagerTest() {
+        try {
+            KryoManager manager = new KryoManager(directoryPath);
+            manager.saveGame(fileName, saved);
+            MapState<GameObject> loaded = manager.loadGame(fileName);
+            assertEquals(saved, loaded);
+        } catch (SaveGameException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    @Test
+    void JacksonManagerTest() {
```

Comment:
Method name &#39;JacksonManagerTest&#39; must match pattern &#39;^[a-z][a-z0-9][a-zA-Z0-9_]*$&#39;.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-548382981>*